### PR TITLE
added validation stringency for loadIndexedBam

### PR DESCRIPTION
**Pull request by akmorrow13**

### CODE REVIEW

**Comment by akmorrow13** [comment]
Im not sure where to find a file to test this

File: adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala [hunk 1]
```diff
@@ -1542,7 +1542,8 @@ class ADAMContext(@transient val sc: SparkContext) extends Serializable with Log
    */
   def loadIndexedBam(
     pathName: String,
-    viewRegions: Iterable[ReferenceRegion])(implicit s: DummyImplicit): AlignmentRecordRDD = LoadIndexedBam.time {
+    viewRegions: Iterable[ReferenceRegion],
+    stringency: ValidationStringency = ValidationStringency.STRICT)(implicit s: DummyImplicit): AlignmentRecordRDD = LoadIndexedBam.time {
```
Comment (fnothaft) [inline adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala]: Can you add scaladoc for this?

Comment (fnothaft) [reply adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala]: Also, can you add this to the other definition of `loadIndexedBam`?

Comment (akmorrow13) [reply adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala]: I added a scaladoc. We can't add this to the loadIndexed bam due to the same overloadding issue we had with loadIndexedVcf

Comment (fnothaft) [reply adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala]: Ah, gotcha! SGTM.

**Comment by fnothaft** [comment]
Merged! Thanks @akmorrow13!

**Pull request merged**